source/lac/scalapack.cc

```diff
@@ -75,31 +75,56 @@ ScaLAPACKMatrix<NumberType>::ScaLAPACKMatrix(const size_type n_rows_,
                                              const std::shared_ptr<const Utilities::MPI::ProcessGrid> &process_grid,
                                              const size_type row_block_size_,
                                              const size_type column_block_size_,
-                                             const LAPACKSupport::Property property)
+                                             const LAPACKSupport::Property property_)
   :
-  TransposeTable<NumberType> (),
```

Comment:
frankly not sure about this, but I think you should keep the call to the parent's constructor here.

Reply:
The base class constructor will be called before anything else happens. This does not depend on you explicitly calling it or not. 
On first glance, it looks a bit weird that you don't pass the number of rows and columns to the base class though.

Reply:
> On first glance, it looks a bit weird that you don't pass the number of rows and columns to the base class though.

that's fine because you don't know this yet, remember this is a parallel dense matrix.

Reply:
Ah, this happens in https://github.com/dealii/dealii/pull/6119/files/a1fd596ea4d111973e48aa0893ca5cb1789d5464#diff-49cd533f816dcc95228fe13f530354a2R144.

Reply:
correct